### PR TITLE
SaveSetMutation

### PR DESCRIPTION
**Pull request by cy**
~~wip~~ ready for review

### CODE REVIEW

File: src/packages/@ncigdc/components/Oncogrid/OncogridWrapper.js [hunk 1]
```diff
@@ -235,7 +235,7 @@ const OncoGridWrapper = compose(
       const performanceContext = {
         donors: responses.cases.length,
         genes: responses.genes.length,
-        occurences: responses.occurences.length,
+        occurences: responses.occurrences.length,
```
Comment (shanewilson) [inline src/packages/@ncigdc/components/Oncogrid/OncogridWrapper.js]: key is still `occurences`

File: data/schema.graphql [hunk 2]
```diff
@@ -4810,6 +4861,10 @@ enum Mode {
   sum
 }
 
+type Mutation {
+  sets(input: RelayIsDumb): Sets
```
Comment (cheapsteak) [inline data/schema.graphql]: Hahaha 🤣

File: src/packages/@ncigdc/components/Oncogrid/OncogridWrapper.js (hunk 1 above)
Comment (cheapsteak) [inline src/packages/@ncigdc/components/Oncogrid/OncogridWrapper.js]: Existing metrics were gathered as `occurences`, want to keep the data consistent

Comment (shanewilson) [inline src/packages/@ncigdc/components/Oncogrid/OncogridWrapper.js]: you should change it we have barely any metrics - no reason to leave it wrong forever

File: data/schema.graphql (hunk 2 above)
Comment (cy) [inline data/schema.graphql]: credit to @shane :P

File: src/packages/@ncigdc/components/Oncogrid/OncogridWrapper.js (hunk 1 above)
Comment (cheapsteak) [inline src/packages/@ncigdc/components/Oncogrid/OncogridWrapper.js]: PR opened https://github.com/NCI-GDC/portal-ui/pull/1871/files

**Comment by cy** [comment]
booo wasn't working in one direction, so looked at the relay generated file, and though there are two mutations, only one query got generated and it uses the same one even when the other is specified. :/ Will have to separate into two components.

File: src/packages/@ncigdc/modern_components/SaveSetButton/SaveSetButton.js [hunk 3]
```diff
@@ -31,12 +31,39 @@ const mutation = graphql`
...
 
-const commitSaveSet = ({ filters, onCompleted, onError }) => {
+const repositoryMutation = graphql`
+  mutation SaveSetButtonMutation(
+    $input: CreateSetInput
+    $never_used: RelayIsDumb
+  ) {
+    sets(input: $never_used) {
+      create {
+        repository {
```
Comment (shanewilson) [inline src/packages/@ncigdc/modern_components/SaveSetButton/SaveSetButton.js]: this is the only mutation isn't it?

Comment (shanewilson) [inline src/packages/@ncigdc/modern_components/SaveSetButton/SaveSetButton.js]: oh I was looking at old file

OH you called them the same thing maybe?

**Comment by cy** [comment]
requires https://github.com/NCI-GDC/gdcapi/pull/806

File: src/packages/@ncigdc/components/CurrentFilters.js [hunk 4]
```diff
@@ -81,6 +81,19 @@ type TProps = {
   hideLinkOnEmpty: boolean,
 };
 
+export const getDisplayOp = (op, value) => {
+  if (op === 'in') {
```
Comment (Jephuff) [inline src/packages/@ncigdc/components/CurrentFilters.js]: Probably should toLowerCase the `op` to catch `IN`.

File: src/packages/@ncigdc/containers/explore/ExplorePage.js [hunk 5]
```diff
@@ -153,41 +156,55 @@ export const ExplorePageComponent = (props: TProps) =>
...
-          },
-          {
-            id: 'oncogrid',
-            text: 'OncoGrid',
-            component: <OncogridTab />,
-          },
-        ]}
-      />
+      <span>
+        <Row
```
Comment (Jephuff) [inline src/packages/@ncigdc/containers/explore/ExplorePage.js]: This row isn't doing anything besides the padding right? Could remove it and apply the padding to the button.

**Pull request merged**